Tests/requires_internet.py

```
@@ -6,29 +6,53 @@
-            check.available = True
+            results.append(True)
+
+    thread = threading.Thread(target=probe, daemon=True)
+    thread.start()
+    thread.join(timeout)
+    return bool(results) and results[0]
+
+
+def check():
+    try:
+        check.available
```

Ruff (0.16.1)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)